### PR TITLE
9/16 Email

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2022-09-16.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub.
+
+I can't believe [I signed up to write a joke per week](https://mailchi.mp/dolthub.com/tims-weekly-dolthub-update-n5f023c6xj-9162255). I'm not funny. Being funny weekly is the hardest part of this email. 
+
+### Dolt Performance
+
+Dolt is getting faster all the time. [Andy](https://www.dolthub.com/team#andy) has been busy building the [new, faster, storage format](https://www.dolthub.com/blog/2022-05-20-new-format-alpha/) all year. [Max](https://www.dolthub.com/team#andy) has been grinding away [making improvements to our SQL analyzer](https://www.dolthub.com/blog/2022-09-14-index-join-perf/).
```
Comment (tbantle22) [inline emails/2022-09-16.md:7]:
```suggestion
Dolt is getting faster all the time. [Andy](https://www.dolthub.com/team#andy) has been busy building the [new, faster, storage format](https://www.dolthub.com/blog/2022-05-20-new-format-alpha/) all year. [Max](https://www.dolthub.com/team#max) has been grinding away [making improvements to our SQL analyzer](https://www.dolthub.com/blog/2022-09-14-index-join-perf/). 
```

File: emails/2022-09-16.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Also, [the data](https://www.dolthub.com/repositories/dolthub/us-housing-prices-v2) is licensed Creative Commons. Get your analysis started! We're always happy to have guest data analysis posts on [our blog](https://www.dolthub.com/blog/) if you're trying to build a portfolio.
+
+### go-mysql-server
+
+[go-mysql-server](https://github.com/dolthub/go-mysql-server) is the other open source project we support. It's fairly popular in its own right with almost 1,000 GitHub stars. `go-mysql-server` is a storage agnostic, MySQL compatible SQL engine. [Zach wrote about go-mysql-server's interface](https://www.dolthub.com/blog/2022-09-12-golang-interface-extension/) this week.
+
+We `go-mysql-server` to power Dolt so it's very full-featured, supporting indexes, foreign keys, constraints, among other SQL features. Most people use it to write MySQL integration tests that don't require a running MySQL. We don't intend to monetize `go-mysql-server` so if you're a user, throw us a [sponsor](https://github.com/sponsors/dolthub) or at least a GitHub star.
```
Comment (tbantle22) [inline emails/2022-09-16.md:27]:
```suggestion
We use `go-mysql-server` to power Dolt so it's very full-featured, supporting indexes, foreign keys, constraints, among other SQL features. Most people use it to write MySQL integration tests that don't require a running MySQL. We don't intend to monetize `go-mysql-server` so if you're a user, throw us a [sponsor](https://github.com/sponsors/dolthub) or at least a GitHub star.
```

Comment (fulghum) [inline emails/2022-09-16.md:27]: Missing a word at the start there... 

```suggestion
We use `go-mysql-server` to power Dolt so it's very full-featured, supporting indexes, foreign keys, constraints, among other SQL features. Most people use it to write MySQL integration tests that don't require a running MySQL. We don't intend to monetize `go-mysql-server` so if you're a user, throw us a [sponsor](https://github.com/sponsors/dolthub) or at least a GitHub star.
```

**Review by fulghum — APPROVED** [review]
🚀

**Pull request merged**